services/backend/src/tmp/connectivity_pa_timeline_prot_conn.json

```diff
@@ -0,0 +1,26 @@
+{
+  "key": "ProtConn",
+  "label": "Prot-Conn",
```

Comment:
Also, i don't think we should send the label, it can be specified in the frontend